satpy/resample.py

```diff
@@ -109,6 +109,14 @@
     >>> my_area = AreaDefinition(...)
     >>> local_scene = scn.resample(my_area)
 
+Resize area definition in pixels
+--------------------------------
+
+Sometimes you may want to create a small image with fixed size in pixels.
+For example, to create an image of (x, y) pixels :
```

Comment:
Possibly controversial, but maybe this should be `(y, x)` for those used to see numpy array shapes (ex. `(num_rows, num_cols)`)?

Reply:
right